os/dbuild.sh

```diff
@@ -46,6 +47,30 @@ if ! which docker > /dev/null; then
 	exit 1
 fi
 
+# check docker image and pull docker image
+function CHECK_DOCKER_IMAGE()
```

Comment:
I think this function's main operation is getting a specific version.
How about GET_SPECIFIC_DOCKER_IMAGE?

Reply:
Thank you for review, I changed function name.